pom.xml

```diff
@@ -21,7 +21,7 @@
     <java.version>1.7</java.version>
     <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
     <project.reporting.outputEncoding>UTF-8</project.reporting.outputEncoding>
-    <oasp4j.version>2.2.0-SNAPSHOT</oasp4j.version>
+    <oasp4j.version>2.1.1</oasp4j.version>
```

Comment:
what do you mean? I see the PR still shows 2.2.0-SNAPSHOT... where have you changed it?